block/src/Pos/Block/WorkMode.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE DataKinds #-}
+
+-- | Various constraints needed for block processing.
+
+module Pos.Block.WorkMode
```

Comment:
I'd actually rename it into `BlockWorkMode`.

Reply:
Maybe call it simply `Mode`? We have many modules with this name.